### PR TITLE
Fix the black screen after setting the overlay permission.

### PR DESCRIPTION
**Pull request by Sunwoo-Nam**
Create a new setting activity for handling the overlay permission.

### CODE REVIEW

**Review by yg48 — APPROVED** [review]
LGTM. 👍

File: chrome/android/chrome_java_sources.gni [hunk 1]
```diff
@@ -1830,6 +1830,7 @@ chrome_java_sources = [
 
 chrome_java_sources += [
   "java/src/org/chromium/chrome/browser/AlwaysOnTopService.java",
+  "java/src/org/chromium/chrome/browser/CastanetsSettings.java",
   "java/src/org/chromium/chrome/browser/firstrun/CastanetsFragment.java",
```
Comment (Chikdolman) [inline chrome/android/chrome_java_sources.gni:1834]: Should this be added only when Castanets flag is on?

File: chrome/android/chrome_java_sources.gni [hunk 2]
```diff
@@ -1830,6 +1830,7 @@ chrome_java_sources = [
 
 chrome_java_sources += [
   "java/src/org/chromium/chrome/browser/AlwaysOnTopService.java",
+  "java/src/org/chromium/chrome/browser/CastanetsSettings.java",
```
Comment (Chikdolman) [inline chrome/android/chrome_java_sources.gni:1833]: Should this be added only when Castanets flag is on?

**Review by Chikdolman — APPROVED** [review]
LGTM we can refactor later

File: chrome/android/chrome_java_sources.gni (hunk 2 above)
Comment (Sunwoo-Nam) [reply chrome/android/chrome_java_sources.gni:1833]: You're right but we should use dynamic loading for solving this problem.
I'm going to refactor later... :)

**Pull request merged**